{"name": "TC3_004: Verify successful login with invalid data via API", "status": "passed", "steps": [{"name": "Step 1: Send a POST request to /users/login", "status": "passed", "attachments": [{"name": "API Response", "source": "c53243a3-e742-4533-ad3b-32f1f2f6cd1f-attachment.json", "type": "application/json"}], "start": 1736758033415, "stop": 1736758034488}], "attachments": [{"name": "log", "source": "6a6a25e1-55a6-490a-a959-f673cac5b1a7-attachment.txt", "type": "text/plain"}], "start": 1736758033415, "stop": 1736758034488, "uuid": "d7f5ddf5-c466-4223-a87d-ae728135d385", "historyId": "a2e24d3ae71a71bded3f9b0fe945fbb7", "testCaseId": "a2e24d3ae71a71bded3f9b0fe945fbb7", "fullName": "tests.api.test_login_via_api.TestUserLoginFunctionality#test_failed_login_via_api", "labels": [{"name": "parentSuite", "value": "Regression"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "User Login Functionality"}, {"name": "suite", "value": "User Login Functionality"}, {"name": "subSuite", "value": "TC3_004: Verify successful login with invalid data via API"}, {"name": "tag", "value": "tags('API_login_functionality')"}, {"name": "host", "value": "CHEQ-J7M1HB3"}, {"name": "thread", "value": "14480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_login_via_api"}]}